Mathlib/Topology/Algebra/InfiniteSum/NatInt.lean

```diff
@@ -321,17 +259,236 @@ end UniformGroup
 
 section TopologicalGroup
 
-variable {G : Type*} [TopologicalSpace G] [AddCommGroup G] [TopologicalAddGroup G] {f : α → G}
+variable [TopologicalSpace α] [AddCommGroup α] [TopologicalAddGroup α]
```

Comment:
I would perhaps argue in favor of calling groups `G` (or maybe `A` when they are additive) consistently, rather than using the pretty generic `α`. But doing this everywhere would be a lot of work... (and my feelings are not particularly strong).